frontend/src/features/admin-form/responses/ResponsesPage/storage/UnlockedResponses/hooks/usePageSearchParams.ts

```diff
@@ -17,12 +19,35 @@ export const usePageSearchParams = () => {
       if (page < 0) {
         page = 1
       }
-      setParams({
-        [PAGE_KEY]: String(page),
-      })
+      params.set(PAGE_KEY, page.toString())
+      setParams(params)
```

Comment:
Why do we no longer clear the whole query string and only leave `PAGE_KEY`, like was done previously?

Reply:
in case there are other values in the query, just in case they get cleared accidentally. This hook does not actually care about the other params, just that the page param is set properly.

Reply:
Got it. Thanks!